packages/engine-server/src/markdown/remark/backlinks.ts

```diff
@@ -0,0 +1,49 @@
+import { NoteUtilsV2 } from "@dendronhq/common-all";
+import { MDUtilsV4 } from "@dendronhq/engine-server";
+import { Root } from "mdast";
+import { paragraph } from "mdast-builder";
+import Unified, { Plugin } from "unified";
+import { Node } from "unist";
+import u from "unist-builder";
+
+// Plugin that adds backlinks at the end of each page if they exist
+const plugin: Plugin = function (this: Unified.Processor) {
+  const proc = this;
+  function transformer(tree: Node) {
+    let root = tree as Root;
+    const { fname, vault } = MDUtilsV4.getDendronData(proc);
+    const { engine } = MDUtilsV4.getEngineFromProc(proc);
+    const note = NoteUtilsV2.getNotesByFname({
```

Comment:
should use `getNoteByFnameV5` for getting a single note. `getNotesByFname` is for when you only have the file name